cmd/xeniad/routes/routes.go

```diff
@@ -127,6 +118,7 @@ func routes(a *app.App) {
 
 	a.Handle("POST", "/v1/exec", handlers.Exec.Custom)
```

Comment:
As part of this PR, please also extend the Custom exec endpoint like so:

```
a.Handle("POST", "/v1/exec/view/:view/:item", handlers.Exec.CustomOnView)
```

Until we can figure out the best ways to save query_sets safely and promote them across environments all of the development and testing is done via /v1/exec. Initially, we will be exclusively be using the CustomOnView functionality for initial development.